contracts/DCAHub/DCAHubPositionHandler.sol

```diff
@@ -23,7 +23,8 @@ abstract contract DCAHubPositionHandler is ReentrancyGuard, DCAHubConfigHandler,
   IDCAPermissionManager public permissionManager;
   mapping(uint256 => DCA) internal _userPositions;
   mapping(uint256 => uint256) internal _swappedBeforeModified;
-  uint256 internal _idCounter;
+  /// @inheritdoc IDCAHubPositionHandler
+  uint256 public idCounter;
```

Comment:
Nit: since we are making it public, maybe we should call it `totalCreatedPositions`, or something like that? I think it would be a little clearer